.github/workflows/integration-tests.yml

```diff
@@ -33,9 +33,10 @@ jobs:
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
       - name: Install uv
-        uses: astral-sh/setup-uv@0c5e2b8115b80b4c7c5ddf6ffdd634974642d182 # v5.4.1
+        uses: astral-sh/setup-uv@c7f87aa956e4c323abf06d5dec078e358f6b4d04 # v6.0.0
         with:
           python-version: "3.10"
+          activate-environment: true
```

Comment:
Is this a new thing with v6? Why is this the only action being updated?

Reply:
Yes, and the default is false. I think other workflows activate the environment manually.